recipes-nodes/orderflow-proxy/files/orderflow-proxy.conf.mustache

```diff
@@ -2,5 +2,5 @@ export FLASHBOTS_ORDERFLOW_SIGNER_ADDRESS={{orderflow_proxy.flashbots_orderflow_
 export LOCAL_LISTEN_ADDR={{orderflow_proxy.local_listen_addr}}
 export PUBLIC_LISTEN_ADDR={{orderflow_proxy.public_listen_addr}}
 export BUILDER_ENDPOINT={{orderflow_proxy.builder_endpoint}}
-export CERT_HOSTS={{orderflow_proxy.cert_hosts}}
+export CERT_HOSTS=localhost,127.0.0.1,{{public_ip}},{{dns_name}}
```

Comment:
why limit it to these, instead of the full list of `cert_hosts` received from BuilderHub?

Reply:
Upstream change in BuilderHub separated these into distinct top-level keys. Both IP address and DNS name separately may come in handy for other services inside the image.